### DISCUSSION TITLE
github codespace not working

### DISCUSSION THREAD

**Discussion by ChristianSantanaLlanes** [community/community#35975] [Codespaces] [answered]
github codespace not working

**Comment by thomas-sickert** [comment, marked as answer]
It looks like this was related to a [Codespaces incident](https://www.githubstatus.com/incidents/7hfyz8mjg663). Are you able to connect to codespaces again? 

To check the status of the Codespaces service, you can visit the [GitHub status page](https://www.githubstatus.com/) or check the [@githubstatus twitter account](https://twitter.com/githubstatus).
